recipe/meta.yaml

```
@@ -29,7 +29,9 @@ test:
 
 about:
   home: http://ipython.org
-  license: BSD 3-Clause
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
```

Looking [at upstream]( https://github.com/ipython/ipython_genutils/blob/0.2.0/COPYING.md ), this appears to be the license filename

```suggestion
  license_file: COPYING.md
```

---

Ah, I assumed it referred to the file in the feedstock. Thanks.